src/tests/tracing/eventpipe/processinfo/processinfo.cs

```diff
@@ -224,6 +224,7 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.LoongArch64 => "loongarch64",
```

Comment:
nit: while at it, keep in sync:

```suggestion
                Architecture.LoongArch64 => "loongarch64",
                Architecture.RiscV64 => "loongarch64",
```